pkgs/top-level/all-packages.nix

```diff
@@ -3248,6 +3248,8 @@ with pkgs;
 
   itch = callPackage ../games/itch {};
 
+  itchiodl = callPackage ../games/itchiodl { };
```

Comment:
Is a download tool a, game? Anyway these categories just make no sense sometimes.

Reply:
A download tool, but so are `itch` and `steam`. `minigalaxy` on the other hand is in `applications/misc`, but I decided to go with the majority